mnefun/data/canonical.yml

```diff
@@ -0,0 +1,179 @@
+general:
+  work_dir: '.'
+  subjects_dir:
+  subject_indices:
+  disp_files:
+
+fetch_raw:
+  subjects:
+  structurals:
+  dates:
+  acq_ssh:
+  acq_dir:
+  acq_port:
+  run_names:
+  runs_empty:
+  subject_run_indices:
+
+scoring:
+  score:
+  on_process:
+
+preprocessing:
+  multithreading:
+    n_jobs:
+    n_jobs_mkl:
+    n_jobs_fir:
+    n_jobs_resample:
+  bads:
+    mf_autobad:
+    mf_badlimit:
+    auto_bad:
+    auto_bad_reject:
+    auto_bad_flat:
+    auto_bad_eeg_thresh:
+    auto_bad_meg_thresh:
+  head_position_estimation:
+    coil_t_window:
+    coil_t_step_min:
+    coil_dist_limit:
+  annotations:
+    coil_bad_count_duration_limit:
+    rotation_limit:
+    translation_limit:
+  sss:
+    movecomp:
+    sss_type:
+    int_order:
+    ext_order :
+    sss_regularize:
+    tsss_dur:
+    st_correlation:
+    filter_chpi:
+    trans_to:
+    sss_origin:
+    dig_with_eeg:
+    ct_file:
+    cal_file:
+    sss_format:
+    mf_args:
+  filtering:
+    hp_cut:
+    hp_trans:
+    lp_cut:
+    lp_trans:
+    filter_length:
+    fir_design:
+    fir_window:
+    phase:
+  ssp:
+    proj_nums:
```

Comment:
As is `params = mnefun.read_params(...)` doesn't bring up relevant `docstring` for e.g., order of projections by channel type

Reply:
Indeed currently all docs for `mnefun` params are in https://labsn.github.io/mnefun/overview.html